packages/@headlessui-react/src/index.test.ts

```diff
@@ -6,14 +6,14 @@ import * as HeadlessUI from './index'
  */
 it('should expose the correct components', () => {
   expect(Object.keys(HeadlessUI)).toEqual([
+    'Portal',
     'Combobox',
     'Dialog',
     'Disclosure',
     'FocusTrap',
     'Listbox',
     'Menu',
     'Popover',
-    'Portal',
     'RadioGroup',
```

Comment:
Should be sorted alphabetically? :)

Reply:
They are, but the exports are sorted alphabetically and the export for the portal itself changed which is now hoisted to the top in the test. 

https://github.com/tailwindlabs/headlessui/pull/2492/files#diff-1532aa9678ff2e077efc3e5259963ecd0cb1725bb4dabf53ffabf87662f19594